_posts/2023-09-07-develop-functions-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "The faster way to iterate on your OpenFaaS functions"
+description: "Learn how the local-run and watch features of faas-cli can help you iterate on functions faster."
+date: 2023-09-07
+categories:
+- faas-cli
+- openfaas
+- functions
+- devex
+dark_background: true
+author_staff_member: han
+image: "/images/2023-09-develop-functions-locally/background.png"
```

Comment:
Please change the VScode image from what I put in the Canva that I shared with you.

Here's how:

Open your real code example from this blog post.

Pick a color theme that you like, and that has enough contrast.

Take a partial screenshot or crop a full one.

Add it to Canva on the canvas, then edit it and apply a blur.